test/inductor/test_torchinductor.py

```diff
@@ -87,7 +87,8 @@
 slow = functools.partial(unittest.skipIf, not TEST_WITH_SLOW, "too slow")
 
 config.triton.autotune_pointwise = False  # too slow
-
+# make sure we get errors if we don't have explicit fallback
+config.implicit_fallbacks = False
```

Comment:
Let's move this (and the one above) to a decorator:
```
@config.patch(implicit_fallbacks=False)
```

You could do something like:
https://github.com/pytorch/pytorch/blob/6c555b29a8cf23e99c03887b75af22bf43f52d41/torch/_dynamo/test_case.py#L44-L55

Reply:
Thanks, it's done